test/val/val_capability_test.cpp

```diff
@@ -1292,46 +1333,47 @@ make_pair(string(kOpenCLMemoryModel) +
 // See https://github.com/KhronosGroup/SPIRV-Tools/issues/365
 INSTANTIATE_TEST_CASE_P(BuiltIn, ValidateCapabilityVulkan10,
                         Combine(
-                            // Vulkan 1.0 is based on SPIR-V 1.0
-                            ValuesIn(AllV10Capabilities()),
+                            // All capabilities to try.
+                            ValuesIn(AllSpirV10Capabilities()),
                             Values(
 make_pair(string(kGLSL450MemoryModel) +
           "OpEntryPoint Vertex %func \"shader\" \n" +
           "OpDecorate %intt BuiltIn PointSize\n"
           "%intt = OpTypeInt 32 0\n" + string(kVoidFVoid),
-          AllV10Capabilities()),
+          // Capabilities which should succeed.
+          AllVulkan10Capabilities()),
 make_pair(string(kGLSL450MemoryModel) +
           "OpEntryPoint Vertex %func \"shader\" \n" +
           "OpDecorate %intt BuiltIn ClipDistance\n"
           "%intt = OpTypeInt 32 0\n" + string(kVoidFVoid),
-          AllV10Capabilities()),
+          AllVulkan10Capabilities()),
 make_pair(string(kGLSL450MemoryModel) +
           "OpEntryPoint Vertex %func \"shader\" \n" +
           "OpDecorate %intt BuiltIn CullDistance\n"
           "%intt = OpTypeInt 32 0\n" + string(kVoidFVoid),
-          AllV10Capabilities())
+          AllVulkan10Capabilities())
 )),);
 
 INSTANTIATE_TEST_CASE_P(BuiltIn, ValidateCapabilityOpenGL40,
                         Combine(
                             // OpenGL 4.0 is based on SPIR-V 1.0
-                            ValuesIn(AllV10Capabilities()),
+                            ValuesIn(AllSpirV10Capabilities()),
                             Values(
 make_pair(string(kGLSL450MemoryModel) +
           "OpEntryPoint Vertex %func \"shader\" \n" +
           "OpDecorate %intt BuiltIn PointSize\n"
           "%intt = OpTypeInt 32 0\n" + string(kVoidFVoid),
-          AllV10Capabilities()),
+          AllSpirV10Capabilities()),
 make_pair(string(kGLSL450MemoryModel) +
           "OpEntryPoint Vertex %func \"shader\" \n" +
           "OpDecorate %intt BuiltIn ClipDistance\n"
           "%intt = OpTypeInt 32 0\n" + string(kVoidFVoid),
-          AllV10Capabilities()),
+          AllSpirV10Capabilities()),
 make_pair(string(kGLSL450MemoryModel) +
           "OpEntryPoint Vertex %func \"shader\" \n" +
           "OpDecorate %intt BuiltIn CullDistance\n"
           "%intt = OpTypeInt 32 0\n" + string(kVoidFVoid),
-          AllV10Capabilities())
+          AllSpirV10Capabilities())
 )),);
 
```

Comment:
You've added a check to the validator.  So you need a negative test to demonstrate that something is rejected.